backend/src/main/java/capstone/backend/controller/OrderToSupplierController.java

```diff
@@ -0,0 +1,29 @@
+package capstone.backend.controller;
+
+
+import capstone.backend.model.dto.order.OrderToSupplierDTO;
+import capstone.backend.services.OrderToSupplierService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("api/orders_suppliers")
```

Comment:
Hyphens or Underscore in application? For google crawler the Hyphens is better :)